consoleme/lib/policies.py

```diff
@@ -726,147 +738,210 @@ async def get_url_for_resource(
         region = await get_region_for_arn(arn)
```

Comment:
Nice stuff!

I think we need to handle the situation where `account_id` is `None` here. This function was generally only used for known resources previously. But if we pass in an S3 bucket ARN for an unknown resource, I'd expect this to be None. 

We should try to adhere to the relatively new WebResponse model (This is in consoleme.models) and return a 404 with a description. Weep may need to be updated to show the error form the backend

Reply:
Ah good call, updated the PR to reflect the changes!